daml-lf/spec/daml-lf-1.rst

```diff
@@ -3662,40 +3665,152 @@ Int64 functions
   in ``Some``. If the input does not match the regexp ``[+-]?\d+`` or
   if the result of the conversion overflows, returns ``None``.
 
+MathContext functions
+~~~~~~~~~~~~~~~~~~~~~
+
+*  ``'MATHCTX_UNLIMITED' : MathContext``
+
+   Returns `'UnBondedMathContext'`
+
+* ``MATHCTX_BOUNDED' : RoundingMode -> Int64 -> MathContext``
+
+   Returns ``'BoundedMathContext' v₁ v₂`` where ``v₁`` and ``v₂`` respectively the first and second argument.
```

Comment:
We don't need these functions if we have math context literals.

Reply:
We need it, if we do not want to create a expression for MathContext in the proto.
I am fine with both.

Reply:
Hmm. If you don't want to add a protobuf expression, it sounds like keeping the math context only as built-ins would be more useful? It's not nice for the LF spec to have an expression form with typing rules, etc but you can't read/write in protobuf.

Reply:
It is what we have done for map.
There is expression for maps, but we do not have it in proto.

Reply:
I see, thanks for the clarification :)